appcenter-analytics/scripts/postlink.js

```diff
@@ -1,28 +1,12 @@
 const rnpmlink = require('appcenter-link-scripts');
-const inquirer = require('inquirer');
 
 // Configure Android first.
 let promise = null;
```

Comment:
Nitpick: I think we can remove `return promise` on the last line (45) because it's unused. `promise` should be exported using `module.exports = promise` if we want to get access to it outside of the module.